src/components/Navbar.js

```diff
@@ -72,11 +72,11 @@ export class Navbar extends Component {
                     </strong>
                   </a>
                   <div className="dropdown-menu" aria-labelledby="navbarDropdown" id="navbarDropdown">
-                    <a className="dropdown-item nav-link" href="!#" onClick={this.logoutUser}>
+                    <Link className="dropdown-item nav-link" to="!#" onClick={this.logoutUser}>
```

Comment:
I understand that this might not be part of this ticket but a new ticket has to be created to resolve this. A `Link` should lead to a certain route otherwise use a `button`. Adding a `click` event to a `Link` means its meant to be a button. Visit [this for more context](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md).

Reply:
Thanks for noticing this. I will create a ticket to change this to a button.

Reply:
You're welcome. Please do and add a link to the ticket here.

Reply:
@dondrzzy I created a ticket for this [here](https://issues.openmrs.org/browse/OCLOMRS-261).

Reply:
@kmunene can you make the ticket ready for work if you have described it well enough and feel like it is in position for some one to take it up?

Reply:
Okay. I will do that.